test/parsing/king-11/longName.precomp

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+NAME_MAX=`getconf NAME_MAX /`
+NAME_MAX=$((NAME_MAX - 15))
+LIMIT=$((NAME_MAX - 1))
```

Comment:
Limit is based on `NAME_MAX - 16` which is as per the current selection of 16 which `NAME_MAX - 15` is just out of the limit